conventions/src/main/kotlin/otel.spotless-conventions.gradle.kts

```diff
@@ -76,7 +76,9 @@ spotless {
         // also very hard to find out where this happens
         "ktlint_standard_wrapping" to "disabled",
         // we use variable names like v1_10Deps
-        "ktlint_standard_property-naming" to "disabled"
+        "ktlint_standard_property-naming" to "disabled",
+        // prevent moving comment to next line in latestDepTestLibrary("xxx") { // see xxx module
+        "ktlint_standard_function-literal" to "disabled"
```

Comment:
without disabling this rule we'd end up with
```
latestDepTestLibrary("xxx") {
  // see xxx module
```
that breaks the script that checks the `latestDepTestLibrary` dependencies